hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+import { render } from "testing/utils";
+
+import { mockTeamMultiple } from "testing/mockData";
+
+import TeamDetail from "pages/TeamDetail/TeamDetail";
+
+describe("User info table", () => {
+    test("renders user info table", () => {
+        const { container } = render(<TeamDetail />);
+        const checkboxes = container.getElementsByClassName("MuiCheckbox-root");
+
+        expect(screen.getByText("Team info")).toBeInTheDocument();
+
+        for (let i = 0; i < mockTeamMultiple.profiles.length; i++) {
```

Comment:
nice job on all the tests! you don't need to change this but you can use the forEach() function here instead of a regular for loop
`mockTeamMultiple.profiles.forEach((profile, index) => {})`
javascript developers don't really use for loops anymore.

Reply:
ah i never knew i could easily access the index in forEach loops. ill do this in the future!